.github/workflows/integrations.yml

```diff
@@ -66,3 +66,14 @@ jobs:
       - name: Run ntp build
         run: |
           ./tests/ci/integration/run_ntp_integration.sh
+  socat:
```

Comment:
do we have a heuristic for how to decide whether new integration tests to GHA vs. codebuild? asking for the purposes of PR #1359.

Reply:
moved aforementioned PR to GitHub Actions.